R/nneo_wrangle.R

```diff
@@ -0,0 +1,108 @@
+#############################################################################################
+#' @title Download and wrangle NEON's sensor-based data products
+
+#' @author Josh Roberti \email{jaroberti87@gmail.com}\cr
+#' Dave Durden\cr
+#' Robert Lee
+
+#' @description Retrieve related, sensor-based datasets at a given NEON site for a custom time period, merged per measurement level and/or variable
+#'
+#' @param site_code (character) a site code. Required.
+#' @param time_start (character) YYYY-MM-DD start day to check for files. Required if \code{is.null(time_end)}
```

Comment:
regarding `\code{}` and others - we're using markdown for docs for this pkg, see https://cran.rstudio.com/web/packages/roxygen2/vignettes/markdown.html for a guide